src/pharmpy/tools/estmethod/tool.py

```diff
@@ -26,46 +26,47 @@ def create_workflow(
     algorithm: str,
     methods: Optional[Union[List[str], str]] = None,
     solvers: Optional[Union[List[str], str]] = None,
-    covs: Optional[Union[List[str], str]] = None,
+    parameter_uncertainty_methods: Optional[Union[List[str], str]] = None,
     results: Optional[ModelfitResults] = None,
     model: Optional[Model] = None,
 ):
     """Run estmethod tool.
 
-    Parameters
-    ----------
-    algorithm : str
-        The algorithm to use (can be 'exhaustive', 'exhaustive_with_update' or 'exhaustive_only_eval')
+     Parameters
+     ----------
+     algorithm : str
+         The algorithm to use (can be 'exhaustive', 'exhaustive_with_update' or 'exhaustive_only_eval')
     methods : list or None
-        List of estimation methods to test. Can be specified as 'all', a list of methods, or
-        None (to not test any estimation method)
-    solvers : list, str or None
-        List of solver to test. Can be specified as 'all', a list of solvers, or None (to
-        not test any solver)
-    covs : list, str or None
-        List of covariance to test. Can be specified as 'all', a list of covs, or None (to
-        not evaluate any covariance)
-    results : ModelfitResults
-        Results for model
-    model : Model
-        Pharmpy model
+         List of estimation methods to test.
+         Can be specified as 'all', a list of estimation methods, or None (to not test any estimation method)
+     solvers : list, str or None
+         List of solver to test. Can be specified as 'all', a list of solvers, or None (to
+         not test any solver)
+     parameter_uncertainty_method : list, str or None
```

Comment:
Missed this in the last review but it should be `parameter_uncertainty_methods` right?

Reply:
Good catch! 🎣